components/FaqAccordian.js

```diff
@@ -1,4 +1,4 @@
-import { useState } from "react";
+import React, { useState } from 'react';
```

Comment:
When I'm working in a React Project, I delete `import React from "react"`.
is it necessary to import?

Reply:
Because 'react' must be in scope when using React

According to the 'react' plugin we were already using. Also I think this is a good idea to have React in the file imported because some editors do not have 2 different file types for javascript and javascriptreact